### ISSUE TITLE
gamescope-session stuck on the logo

### ISSUE THREAD

**Issue by nisel11** [ublue-os/bazzite#903]
### Describe the bug

If I install the system on my laptop the gamescope session will not work, it works in ChimeraOS and HoloISO
![17109571782187985735563116592740](https://github.com/ublue-os/bazzite/assets/96194200/882dbc99-f66f-4975-b82b-9c4005693646)
or without "Verifying installation".

### What did you expect to happen?

A working gamescope session

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
■ostree-unverified-image: docker://ghcr.io/ublue-os/bazzite-deck: latest
Digest: sha256: dc57ef18c5878db01e1202800efc585a29fcd30fc656fbcd128b2be41cb2d8c6
Version: 39.20240319.8 (2024-03-19T21:11:32Z)
Initramfs: "-1 /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"
ostree-unverified-image: docker://ghcr.io/ublue-os/bazzite-deck
: latest Digest: sha256: dc57ef10c5870db01e1202800efc585a29fcd30fc656fbcd128b2be41cb2d8c6 Version: 39.28248319.8 (2824-03-19T21:11:322)
```

### Hardware

Laptop: Acer nitro 5 an515-43
GPU: Vega 8 and RX560X
CPU: Amd ryzen 5 3550H

### Extra information or context

It worked before, not without problems, but it worked
If run gamescope -- bazzite-steam on desktop mode ALL working
![17109603723763435136230992958471](https://github.com/ublue-os/bazzite/assets/96194200/73f7f2fa-6ad7-4866-aaba-8ed9032ad702)
gamescope log says "Broken pipe"

**Comment by HikariKnight** [comment]
steam itself is failing to start, go into the TTY (CTRL+ALT+F4) and run
`mv ~/.local/share/Steam ~/.local/share/Steam-old`
then reboot

steam will re-initialize, once that is done just transfer your games back from `~/.local/share/Steam-old`

**Comment by massatt212** [comment]
> steam itself is failing to start, go into the TTY (CTRL+ALT+F4) and run `mv ~/.local/share/Steam ~/.local/share/Steam-old` then reboot
> 
> steam will re-initialize, once that is done just transfer your games back from `~/.local/share/Steam-old`

is there a way to fix this, im getting low priority when playing dota with all these hanging.

**Comment by HikariKnight** [comment]
as i said in my reply, its an issue where steam has initialized wrong and it has to be __manually fixed__ the way i said in the message.
Because it is steam itself, nothing we can do to prevent it because all we do is start steam.
